packages/web-components/fast-components/src/accordion-item/accordion-item.styles.ts

```diff
@@ -27,13 +28,10 @@ import { heightNumber } from "../styles/size";
  * Styles for AccordionItem
  * @public
  */
-export const accordionItemStyles: (
-    context: ElementDefinitionContext,
-    definition: AccordionItemOptions
-) => ElementStyles = (
-    context: ElementDefinitionContext,
-    definition: AccordionItemOptions
-) =>
+export const accordionItemStyles: FoundationElementTemplate<
+    ElementStyles,
+    AccordionItemOptions
+> = (context: ElementDefinitionContext, definition: AccordionItemOptions) =>
```

Comment:
The parameter types are provided by the `FoundationElementTemplate` generics, so `context` and `definition` don't need to be explicitly typed:

```suggestion
export const accordionItemStyles: FoundationElementTemplate<
    ElementStyles,
    AccordionItemOptions
> = (context, definition) =>
```

This also allows us to remove the `ElementDefinitionContext` and `FoundationElementDefinition` type imports where present.